pom.xml

```diff
@@ -55,6 +63,7 @@
     <jaxb22.version>2.2.11</jaxb22.version>
     <!-- 2.3.3b02 does not work with the plugin anymore on Java 9+ -->
     <jaxb23.version>2.3.3</jaxb23.version>
+    <jaxb30.version>3.0.0</jaxb30.version>
     <jaxb-api.version>2.3.1</jaxb-api.version>
```

Comment:
Should we separate this in jaxb2-api and jaxb3-api ?

Reply:
Sounds reasonable. I will fix that.